sf-pcap-ng.c

```diff
@@ -1392,8 +1392,8 @@ pcap_ng_next_packet(pcap_t *p, struct pcap_pkthdr *hdr, u_char **data)
 		frac /= ps->ifaces[interface_id].tsresol;
 		break;
 	}
-	hdr->ts.tv_sec = sec;
-	hdr->ts.tv_usec = frac;
+	hdr->ts.tv_sec = (long) sec;
```

Comment:
This should be done differently on Windows and UN\*X.  b5f5ae1c650c701a70e4cdc0d88ba7767a564820 should do this correctly.